src/FSharp.Configuration/YamlConfigProvider.fs

```diff
@@ -399,11 +399,23 @@ type Root () =
         Serializer settings
 
     let mutable lastLoadedFrom = None
-    
+
+    let errorEvent = new Event<Exception>()
+
     /// Load Yaml config as text and update itself with it.
-    member x.LoadText (yamlText: string) = Parser.parse yamlText |> Parser.update x
+    member x.LoadText (yamlText: string) =
+      try
+        Parser.parse yamlText |> Parser.update x
+      with e ->
+        async { errorEvent.Trigger e } |> Async.Start
```

Comment:
Because you suggested that it be done that way in the comments to issue 70.  

![capture](https://cloud.githubusercontent.com/assets/483712/12428918/6cf33ece-beb5-11e5-8d7c-5cda45826ad2.PNG)

The event, when triggered in the LoadAndWatch, is coming from a separate background thread, so perhaps your intention was to have this behave the same?